examples/fault-tolerance/README.md

```diff
@@ -0,0 +1,8 @@
+# Quarkus Retry
```

Comment:
Sollen wir das dann evtl. auch gleich` Quarkus Fault Tolerance` nennen?

Und unten _how to efficiently test fault tolerance related code in Quarkus applications_

Reply:
Da hast du recht, das ist noch ein überbleibsel :-)